### ISSUE TITLE
Changes from #2424 broke brython-options pythonpath

### ISSUE THREAD

**Issue by CookiePLMonster** [brython-dev/brython#2428]
Due to the changes in https://github.com/brython-dev/brython/commit/db1e1d57d83b96e941a34f11cb8bfdd034680c62, adding a new pythonpath via the following doesn't work as expected anymore:
```html
<brython-options pythonpath="/assets/brython" cache="true" ></brython-options>
```

Before the change, this added `/assets/brython` from the root of the site to PYTHONPATH. With this change, it seems that the path became relative to the HTML file containing the currently executed Python script:
```
brython.js:5736 
          
       GET https://cookieplmonster.github.io/bonuscodes/cmr-3//assets/brython/generators.py 404 (Not Found)
brython.js:5741 Trying to import generators, not found at url https://cookieplmonster.github.io/bonuscodes/cmr-3//assets/brython/generators.py
brython.js:5736 
            
       GET https://cookieplmonster.github.io/bonuscodes/cmr-3//assets/brython/generators/__init__.py 404 (Not Found)
brython.js:5741 Trying to import generators, not found at url https://cookieplmonster.github.io/bonuscodes/cmr-3//assets/brython/generators/__init__.py
brython.js:5736 
    
       GET https://raw.githubusercontent.com/brython-dev/brython/master/www/src/Lib/site-packages/generators.py 404 (Not Found)
brython.js:5741 Trying to import generators, not found at url https://raw.githack.com/brython-dev/brython/master/www/src/Lib/site-packages/generators.py
brython.js:5736       
        
       GET https://raw.githubusercontent.com/brython-dev/brython/master/www/src/Lib/site-packages/generators/__init__.py 404 (Not Found)
brython.js:5741 Trying to import generators, not found at url https://raw.githack.com/brython-dev/brython/master/www/src/Lib/site-packages/generators/__init__.py
brython.js:11017 Traceback (most recent call last):
  File "bonuscodes/cmr-3/", line 2, in <module>
    from generators import cmr3
ModuleNotFoundError: generators
```

You can see the breakage live here: https://cookieplmonster.github.io/bonuscodes/cmr-3/. I will shortly update it with the below workaround to avoid inconveniencing my users, but hopefully with the above steps it's easy to reproduce locally.

Reverting to `brython.js` from https://github.com/brython-dev/brython/blob/3092dfb40bad5a64b4493bf983c08524be75eabf/www/src/brython.js locally resolves the issue. Switching my `pythonpath` options to use the absolute path **does** help too, but is that necessary? Or were paths relative to root (e.g. `/my/path`) never supposed to work to begin with?

**Comment by CookiePLMonster** [comment]
I pushed the workaround live and using an absolute URL, so the online app does not exhibit this issue anymore.

**Comment by PierreQuentel** [comment]
I was almost sure that this fix for #2424 would break something...
Thanks for the report @CookiePLMonster !

**Comment by CookiePLMonster** [comment]
I can confirm the bug is now fixed, thank you for a rapid response!
